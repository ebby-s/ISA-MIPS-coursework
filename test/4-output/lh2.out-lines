TB : INFO : register_v0=0000579d
TB : finished; active=0
