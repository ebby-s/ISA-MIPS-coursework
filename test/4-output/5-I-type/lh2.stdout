Loading test/2-binary/5-I-type/lh2.hex.txt into RAM
VCD info: dumpfile mips_cpu_bus_tb.vcd opened for output.
TB : INFO : RAM_ACCESS: Read from 0xbfc00000, data: 0x2694579d
TB : INFO : RAM_ACCESS: Read from 0xbfc00004, data: 0x26100bb8
TB : INFO : RAM_ACCESS: Read from 0xbfc00008, data: 0x0014a400
TB : INFO : RAM_ACCESS: Read from 0xbfc0000c, data: 0xae140000
TB : INFO : RAM_ACCESS: Read from 0xbfc00010, data: 0x86020002
TB : INFO : RAM_ACCESS: Write to 0x00000bb8, data: 0x579d0000
TB : INFO : RAM_ACCESS: Read from 0xbfc00014, data: 0x0000f808
TB : INFO : RAM_ACCESS: Read from 0x00000bb8, data: 0x579d0000
TB : INFO : RAM_ACCESS: Read from 0xbfc00018, data: 0x00000000
TB : INFO : register_v0=0000579d
TB : finished; active=0
